config.yaml

```diff
@@ -41,4 +37,12 @@ options:
   userid-claim:
     type: string
     default: 'email'
-    description: OpenID Connect claim whose value will be used as the userid.
\ No newline at end of file
+    description: OpenID Connect claim whose value will be used as the userid.
+  public-url:
+    type: string
+    default: ''
+    description: |
+      DEPRECATED - Please leave empty. This configuration option will be removed soon.
+      It has been preserved to avoid breaking compatibility with existing deployments.
+      If the OIDC provider URL has to be changed, please check dex-auth's issuer-url.
+      Publicly-accessible endpoint for cluster
```

Comment:
nit: Is there a reason we moved this to the end of the file?

Reply:
Yes, I wanted to keep it at the bottom as it is deprecated. I guess it makes it more obvious. Do you think we should keep it where it was?